barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/goblin_ultra_circuit_builder.hpp

```diff
@@ -42,6 +42,7 @@ template <typename FF> class GoblinUltraCircuitBuilder_ : public UltraCircuitBui
     void populate_ecc_op_wires(const ecc_op_tuple& in);
     ecc_op_tuple decompose_ecc_operands(uint32_t op, const g1::affine_element& point, const FF& scalar = FF::zero());
     void set_goblin_ecc_op_code_constant_variables();
+    void create_calldata_lookup_gate(const databus_lookup_gate_<FF>& in);
```

Comment:
Moving this to the `private` access specifier to enforce the use of `read_calldata` as the interface. This method is called from within `read_calldata`.